plugins/chat/spec/services/chat/channel_view_builder_spec.rb

```diff
@@ -76,6 +78,11 @@
       )
     end
 
+    it "updates the channel membership last_viewed_at" do
+      channel.membership_for(current_user).update!(last_viewed_at: 1.day.ago)
+      expect { result }.to change { channel.membership_for(current_user).last_viewed_at }
+    end
```

Comment:
Same comment than the one in `update_user_last_read_spec`, I might be missing something but this test would pass without calling result.